Documentation/cmdref/cilium_service_update.md

```diff
@@ -19,6 +19,7 @@ cilium service update [flags]
       --frontend string    Frontend address
   -h, --help               help for update
       --id uint            Identifier
+      --k8s-external       Set service as a k8s external IP
```

Comment:
I don't think this nor the daemon flag should have any k8s mention in it, so lets do just `--external-ip`. Neither struct members in the BPF datapath since the concept itself is generic and could potentially be used outside of k8s w/o any problem. So I think it's better to remove all the k8s mentions in the datapath and agent/cli flags. We have this similarly with nodeport etc.

Reply:
@borkmann I've mentioned this in the previous comments, the mention of k8s is on purpose to distinguish from potential future work that involves a type of IP that is external to the cluster. Although the concept itself is generic the functionality is tightly couple to a k8s feature.

Reply:
Ok, is there anything coming up from k8s you're aware of that would have this potential naming clash?

Reply:
Discussed offline